apps/web/components/pages/task/details-section/blocks/task-estimations-info.tsx

```diff
@@ -23,7 +23,7 @@ const TaskEstimationsInfo = () => {
 			>
 				<Disclosure>
 					{({ open }) => (
-						<div className="flex flex-col w-full">
+						<div className="flex flex-col w-full mt-[0.1875rem]">
```

Comment:
<img width="527" alt="image" src="https://github.com/ever-co/ever-teams/assets/81486442/e3c6abf7-99a9-4992-9897-f4f01d7e917b">

@desperado1802 this is still not properly centered.